examples/measure.html

```diff
@@ -1,10 +1,12 @@
 ---
 template: example.html
 title: Measure example
-shortdesc: Example of using the ol.interaction.Draw interaction for creating simple measuring application.
+shortdesc: Example of using the ol.interaction.Draw interaction for creating a simple measuring application.
 docs: >
-  <p><i>NOTE: If use geodesic measures is not checked, measure is done in simple way on projected plane. Earth curvature is not taken into account</i></p>
-tags: "draw, edit, measure, vector"
+  <p><i>NOTE: Area and length calculations are done using ol.proj.getLength and ol.proj.getArea to get answers in meters or square meters at any latitude.
+  The ol.geom.Linestring.getLength method does not give a length in meters for EPSG:3857 or EPSG:4326 linestrings.
```

Comment:
Actually converting back from Haversine metres to degrees is going to be tricky for spheres other than the Earth. There is currently only one value for ol.proj.METERS_PER_UNIT[ol.proj.Units.DEGREES] which is for the Earth (ol.sphere.NORMAL.radius), and used by ol.proj.Projection.prototype.getMetersPerUnit